modules/statics/src/networks.ts

```diff
@@ -154,13 +154,17 @@ class AdaTestnet extends Testnet implements AdaNetwork {
 class Apt extends Mainnet implements AccountNetwork {
   name = 'Apt';
   family = CoinFamily.APT;
-  explorerUrl = 'https://explorer.aptoslabs.com/?network=mainnet';
+  explorerUrl = 'https://explorer.aptoslabs.com/txn/?network=mainnet';
```

Comment:
```suggestion
  explorerUrl = 'https://explorer.aptoslabs.com/txn?network=mainnet';
```

Are you sure the last forward slash is needed?

Reply:
Is this one of the valid url : https://explorer.aptoslabs.com/txn/0xf20205bd0756df9b851db4b7e910e2f4b418afdcf63b41199a7b30e45d6e68b5?network=testnet

Reply:
Have you tested this out locally?

Reply:
Locally server is causing some memory issue.
But using the BG-UI code and inputs here, have tested here and its working.
![image](https://github.com/user-attachments/assets/dbeb22be-1308-4cee-8808-9fd0f4e7b3ef)